newrelic/api/transaction.py

```diff
@@ -566,6 +566,9 @@ def __exit__(self, exc, value, tb):
             for ml_model, version in self._ml_models:
                 self.record_custom_metric(f"Supportability/Python/ML/{ml_model}/{version}", 1)
 
+        if os.environ.get("FUNCTIONS_WORKER_RUNTIME", None):
+            self.record_custom_metric("Supportability/Python/AzureFunctionMode/Enabled", 1)
```

Comment:
Was there a reason why this supportability metric was being recorded once per transaction vs. once when we [connect to the data collector](https://github.com/newrelic/newrelic-python-agent/blob/main/newrelic/core/application.py#L530)?